byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -49,6 +49,57 @@ \subsubsection{Delegation interface rules}
 \label{sec:delegation-interface-rules}
 
 \begin{figure}[htb]
+  \begin{equation}
+    \label{eq:rule:delegation-interface}
+    \inference
+    {
+      {\left(\begin{array}{l}
+       \mathcal{K}\\
+        e\\
+        s
+      \end{array}\right)}
+      \vdash
+      \trans{sdeleg}{}
```

Comment:
Here you could use

```latex
      \trans{\hyperref[eq:sdeleg-bootstrap]{sdeleg}}{}
```

So that a hyper reference to the `SDELEG` rule gets generated.

Reply:
done